pipeline/ephys.py

```diff
@@ -133,15 +149,6 @@ class UnitTrial(dj.Part):
         -> experiment.SessionTrial
         """
 
-    class UnitPosition(dj.Part):
-        definition = """
-        # Estimated unit position in the brain
-        -> master
-        -> ccf.CCF
```

Comment:
We discussed the possibility of making a new `computed` table for convenient querying, but didn't settle on that, because that info can be obtained with just a query (may be a long one). So having a dedicated computed table is still debatable.